src/sign.builds

```diff
@@ -10,8 +10,26 @@
 
   <UsingTask AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" TaskName="ReadSigningRequired" />
 
+  <PropertyGroup Condition="'$(OSGroup)' == 'Windows_NT' and '$(SkipSigning)' != 'true'">
+    <NativePlatform Condition="'$(Platform)' != 'AnyCPU'">$(Platform)</NativePlatform>
+    <NativePlatform Condition="'$(Platform)' == 'AnyCPU'">x64</NativePlatform>
+    <CLRCompressionLocation>$(BinDir)\$(OSGroup).$(NativePlatform).$(ConfigurationGroup)\native\clrcompression.dll</CLRCompressionLocation>
+    <AOTCLRCompressionLocation>$(BinDir)\$(OSGroup).$(NativePlatform).$(ConfigurationGroup)\native_aot\clrcompression.dll</AOTCLRCompressionLocation>
+  </PropertyGroup>
+  
+  <Target Name="GenerateSignForWindowsNative" 
+          Condition="'$(OSGroup)' == 'Windows_NT' and '$(SkipSigning)' != 'true' and Exists('$(CLRCompressionLocation)') and Exists('$(AOTCLRCompressionLocation)')"> 
+    <WriteSigningRequired AuthenticodeSig="$(AuthenticodeSig)"
```

Comment:
Do you know what values are going into these Sig's? We need to be sure that we use what ever marker value that StrongNameSig needs to make sure no strong naming happens.